right/src/mouse_controller.c

```diff
@@ -245,7 +245,7 @@ static void recalculateCurrentSpeed(float x, float y) {
         static uint32_t lastUpdate = 0;
         uint32_t elapsedTime = CurrentTime - lastUpdate;
         float distance = sqrt(x*x + y*y);
-        currentSpeed = distance * 1000.0f / elapsedTime;
```

Comment:
There is basically nothing wrong with doing this, except that it changes interpretation of the values and deprecates the comments. 

I.e., for sake of other people's reading abilities, it might be a good idea to update the comments on lines 240 and 257 to read `px/ms` instead of `px/s`. (Of course I do not care much. I am not especially bad at figuring out meaning of my own code.)

Reply:
Thanks for the heads-up! Just updated the units.